### PR TITLE
Change permissions for integration-tests job on circleci.

### LINKED ISSUE #5657: Integration test for uploading an XPI to DevHub is flaky
The CircleCI integration tests for uploading an XPI to DevHub times out a lot. We should fix it or disable the test.

Example https://circleci.com/gh/mozilla/addons-frontend/8848?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
______ ERROR at setup of test_devhub_addon_upload[Resolution: 1080x1920] _______
...
E       TimeoutException: Message:
```

### PR DESCRIPTION
**Pull request by jrbenny35**
Fixes #5657

### CODE REVIEW

**Comment by jrbenny35** [comment]
I don't think we can run this without merging but this _should_ fix it.

File: tests/ui/run-functional-tests.sh [hunk 1]
```diff
@@ -20,4 +20,5 @@ docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.fu
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart worker
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec web make -f Makefile-docker update_deps
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart web
-docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec --user root selenium-firefox tox -e ui-tests
+sudo chown -R circleci:circleci .
```
Comment (kumar303) [inline tests/ui/run-functional-tests.sh]: The old command was running as root (`docker-compose ... exec --user root`) which would have already had access, right? So how would this fix it? Maybe I'm misunderstanding what `--user root` does.

Also, for a change like this it helps to put a comment above saying that it's needed for uploading XPI files (or something like that).

Comment (kumar303) [reply tests/ui/run-functional-tests.sh]: Huh, ok

File: tests/ui/run-functional-tests.sh [hunk 2]
```diff
@@ -20,4 +20,6 @@ docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.fu
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart worker
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec web make -f Makefile-docker update_deps
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart web
-docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec --user root selenium-firefox tox -e ui-tests
+# This is done because cirlceci has permissions do not allow the user `circleci` to remove or add files created by the root, even if it is part of the root group.
```
Comment (kumar303) [inline tests/ui/run-functional-tests.sh]: typo: has permissions to not allow...

**Pull request merged**